pmd-java/src/main/java/net/sourceforge/pmd/lang/java/typeresolution/ClassTypeResolver.java

```diff
@@ -6,7 +6,7 @@
 
 import static net.sourceforge.pmd.lang.java.typeresolution.MethodTypeResolution.getApplicableMethods;
 import static net.sourceforge.pmd.lang.java.typeresolution.MethodTypeResolution.getBestMethodReturnType;
-import static net.sourceforge.pmd.lang.java.typeresolution.MethodTypeResolution.getMethodExplicitTypeArugments;
+import static net.sourceforge.pmd.lang.java.typeresolution.MethodTypeResolution.getMethodExplicitTypeArguments;
```

Comment:
Please don't edit anything in this package and subpackages. Everything is deprecated and will be removed in #2689. The base branch for that pr (the grammar development branch) also edits these files to make them compilable with the new grammar. It's not worth the merge conflicts

Reply:
@oowekyala here the package you mean is "net.sourceforge.pmd.lang.java.typeresolution"?

Reply:
@oowekyala done.